javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,91 @@
+function getFullname(firstname, surname, useFormalName){
+    
+    let fullname = (useFormalName == true) ? ("Lord" + "  "+firstname +"  "+ surname) :(firstname +"  "+ surname);
+
+    console.log(fullname);
+
+}
+getFullname('kamal', 'sharma', true);
+getFullname('kamal', 'sharma', false);
+
+// weather wear
+
+function  clothMatch(temperature){
+    if(temperature>=18){
+        console.log("shorts and a t-shirt");
+    }
+    else{
+        console.log("remember a jacket");
+    }
+}
+clothMatch(19); // execute in if condition
+clothMatch(10); // execute in else condition
+
+//Event application
+
+let todayDate = new Date(); // displayed month, date, year and time
+let todayCount = todayDate.getDay(); // counts the number of todayDate
+function getEventWeekday(totalDays){
+let daysForEvent= todayCount + totalDays;
+let week = ["sday", "mday", "tuday", "wed", "Thday", "friday", "satday"];
+
+if (daysForEvent >= 7){
+    daysForEvent = daysForEvent % 7;
+}
+return week[daysForEvent];
```

Comment:
How are we returning, when we are not inside of a function? 😱, We can't do that. Only functions can return something

Reply:
This is in inside of the fucrtion getEventWeekday